test/graphql/mutations/associate_profiles_test.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class AssociateSystemsMutationTest < ActiveSupport::TestCase
```

Comment:
This should fix the tests:

```suggestion
class AssociateProfilesMutationTest < ActiveSupport::TestCase
```